### PR TITLE
Added SmartFace Station into LFIS deployment pattern

### PR DESCRIPTION
**Pull request by jberes1987**

### CODE REVIEW

**Review by Luke-1988 — APPROVED** [review]

File: sf-docker/LFIS/docker-compose.yml [hunk 1]
```diff
@@ -97,6 +97,40 @@ services:
     volumes:
       - "./iengine.lic:/etc/innovatrics/iengine.lic"
 
+  sf-station:
```
Comment (blaskoa) [inline sf-docker/LFIS/docker-compose.yml:100]: don't we want these changes also in jseton-LFIS?

Comment (jberes1987) [reply sf-docker/LFIS/docker-compose.yml:100]: the Jetson does not have LFIS deployment pattern in the current GitHub

Comment (blaskoa) [reply sf-docker/LFIS/docker-compose.yml:100]: wha is this then? https://github.com/innovatrics/smartface/tree/master/sf-docker/special/jetson-LFIS

Comment (jberes1987) [reply sf-docker/LFIS/docker-compose.yml:100]: thank you for pointing that out. It is currently updated for Jetson LFIS as well.

File: sf-docker/special/jetson-LFIS/docker-compose.yml [hunk 2]
```diff
@@ -177,6 +177,39 @@ services:
       - "./iengine.lic:/etc/innovatrics/iengine.lic"
     runtime: nvidia
 
+  sf-station:
+    image: ${REGISTRY}sf-jetson-station:${SFS_VERSION}
```
Comment (blaskoa) [inline sf-docker/special/jetson-LFIS/docker-compose.yml:181]: `${REGISTRY}sf-station:${SFS_VERSION}`

Comment (jberes1987) [reply sf-docker/special/jetson-LFIS/docker-compose.yml]: thank you, fixed.

**Review by blaskoa — APPROVED** [review]

**Pull request merged**